copy/copy_freebsdsolaris.go

```diff
@@ -0,0 +1,33 @@
+//+build freebsd solaris
+
+package fs
+
+import (
+	"github.com/pkg/errors"
+	"os"
+	"io"
+)
```

Comment:
nit: `github.com/pkg/errors` should be in a separate "group" here (not a blocker)

```suggestion
import (
	"os"
	"io"

	"github.com/pkg/errors"
)
```

Reply:
Are we missing some ci validation for this?

Reply:
Hmm.. perhaps, or the linter doesn't catch it (as it's "optional" to group them)

Reply:
The formatter or goimports should catch it